pkg/operation/common/types.go

```diff
@@ -68,8 +68,6 @@ const (
 	GrafanaTLS = "grafana-tls"
 	// PrometheusTLS is the name of the secret resource which holds the TLS certificate for Prometheus.
 	PrometheusTLS = "prometheus-tls"
-	// LokiTLS is the name of the secret resource which holds the TLS certificate for Loki.
-	LokiTLS = "loki-tls"
```

Comment:
Why has this constant been removed? Isn't it better to use constants for secret names, as is also the case for other secrets managed by the secrets manager?

Reply:
The constant is removed for two reasons: 1) We are working on eliminating the `common` package entirely and rather move the constants either to `v1beta1constants` if needed or to the respective component packages. 2) We don't need this secret name anywhere else other than at the place where it's being generated (it's anyways not the "real" secret name but just the name of the secret configuration, so there is little value of having it).

For now, I only kept the constants of the name in case they are also used by other components, otherwise they are removed or made unexported.